### PR TITLE
setup protractor configuration

### PR DESCRIPTION
**Pull request by kliver98**
Step 29 of [workshop](https://github.com/AgileTestingColombia/workshop-protractor)

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+indent_style = space
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
+indent_size = 2
+
+[*.md]
+indent_size = 4
+trim_trailing_whitespace = false
```
Comment (holgiosalos) [inline .editorconfig]: You should add a break line at the end of the file.

Please check this message: https://automation-bootcamp.slack.com/archives/C017QT1M1DY/p1595723443011200

File: .github/CODEOWNERS.md [hunk 2]
```diff
@@ -0,0 +1 @@
+* @aperdomob @germandavid85 @luigisamurai @yesidbalvin @Scot3004 @santirogu
```
Comment (holgiosalos) [inline .github/CODEOWNERS.md]: This file should only have these names: `* @Valeryibarra @renardete @leonleo997 @holgiosalos`

**Review by holgiosalos — APPROVED** [review]

File: .github/CODEOWNERS.md (hunk 2 above)
Comment (yesid-lopez) [reply .github/CODEOWNERS.md]: agree

File: package.json [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    "protactor"
+  ],
+  "author": "Kliver Daniel <kliver1998@gmail.com> (kliver98)",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/kliver98/protractor-workshop-2020/issues"
+  },
+  "homepage": "https://github.com/kliver98/protractor-workshop-2020#readme",
+  "dependencies": {
+    "protractor": "^7.0.0"
```
Comment (yesid-lopez) [inline package.json]: move as a dev dependency

**Comment by yesid-lopez** [comment]
Can you add a screenshot of the test execution? thx

**Pull request closed without merging**